.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,14 @@
+name: Release Drafter
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: release-drafter/release-drafter@v5
```

Comment:
I've never used that, can't wait to see it in action

Reply:
Neither did I. I copied from another hexo project (cli)